src/Command/ListCommand.php

```diff
@@ -42,11 +64,41 @@ protected function execute(
         InputInterface $input,
         OutputInterface $output
     ) {
-        // @TODO Read plugins from database
+        // @TODO Read plugins from database -> done
```

Comment:
then remove the todo?!? x)

Reply:
Dann würde bei den TODOs drunter aber Kontext fehlen 🙂